pokemongo_bot/cell_workers/seen_fort_worker.py

```diff
@@ -58,13 +61,15 @@ def work(self):
 
                     logger.log("[+] " + str(item_count) + "x " + item_name,
                                "green")
+                    rewards.append(str(item_count) + "x " + item_name)
                 if self.config.recycle_items:
                     recycle_worker = RecycleItemsWorker(self.bot)
                     recycle_worker.work()
 
             else:
                 logger.log("[#] Nothing found.", "yellow")
 
+            self.bot.fire('fort_spinned', rewards=rewards)
```

Comment:
maybe make this an object instead of a string?
like `{'xp':50, 'pokeball':3, 'greatball':1}`